Закончил домашку!

mameevdi opened the pull request.

mameevdi committed:
Закончил домашку!

mameevdi committed:
Закончил домашку!

mameevdi committed:
Закончил домашку!

dmitry-bizin commented on src/Main.java:
```diff
@@ -1,5 +1,55 @@
+        int capital = 15_000;
+        int total = 0;
+        int month = 0;
+        while (total < 2_459_000) {
```
2_459_000 лучше вынести в переменную

dmitry-bizin commented on src/Main.java:
```diff
@@ -1,5 +1,55 @@
+    }
+    public static void task3() {
+        System.out.println("Задача 3"); //Рассчитайте, какая численность населения будет через 10 лет
+        double PopulationSize = 12_000_000;
```
названия переменных с маленькой буквы

dmitry-bizin commented on src/Main.java:
```diff
@@ -1,5 +1,55 @@
+        DecimalFormat df = new DecimalFormat("#");
+        System.out.println("Численность населения равна " + df.format(PopulationSize));
+        for (int i = 1; i <= 10; i++) {
+            var BornInYear = PopulationSize/1000*17;
```
названия переменных с маленькой буквы

dmitry-bizin commented on src/Main.java:
```diff
@@ -1,5 +1,55 @@
+        System.out.println("Численность населения равна " + df.format(PopulationSize));
+        for (int i = 1; i <= 10; i++) {
+            var BornInYear = PopulationSize/1000*17;
+            var DiedInYear = PopulationSize/1000*8;
```
названия переменных с маленькой буквы

dmitry-bizin commented on src/Main.java:
```diff
@@ -1,5 +1,55 @@
+        double PopulationSize = 12_000_000;
+        DecimalFormat df = new DecimalFormat("#");
+        System.out.println("Численность населения равна " + df.format(PopulationSize));
+        for (int i = 1; i <= 10; i++) {
```
i лучше назвать year

dmitry-bizin commented on src/Main.java:
```diff
@@ -1,5 +1,55 @@
+            PopulationSize = PopulationSize+(BornInYear-DiedInYear);
+            System.out.println("На " + i + "-й год население города составило " + df.format(PopulationSize) + " человек. Родились " + df.format(BornInYear) + " человека, умерли " + df.format(DiedInYear)+ " человека.");
+        }
+        if (PopulationSize>12_000_000){
```
12_000_000 в переменную

dmitry-bizin commented on src/Main.java:
```diff
@@ -1,5 +1,55 @@
+            System.out.println("На " + i + "-й год население города составило " + df.format(PopulationSize) + " человек. Родились " + df.format(BornInYear) + " человека, умерли " + df.format(DiedInYear)+ " человека.");
+        }
+        if (PopulationSize>12_000_000){
+            var total = PopulationSize-12_000_000;
```
12_000_000 в переменную

dmitry-bizin commented on src/Main.java:
```diff
@@ -1,5 +1,55 @@
+            System.out.println("Население за 10 лет выросло на " + df.format(total) + " человек");
+        }
+        else {
+            var total = 12_000_000 - PopulationSize;
```
12_000_000 в переменную

mameevdi committed:
Исправил ошибки.
1. Все числа которые могут меняться уду стараться выносить в переменные
2. Имена переменных буду писать с маленькой буквы
3. Название переменных буду давать осмысленные
